README.md

```diff
@@ -1,93 +1,144 @@
 # Mattermost Operator for Kubernetes ![CircleCI branch](https://img.shields.io/circleci/project/github/mattermost/mattermost-operator/master.svg) [![Community Server](https://img.shields.io/badge/Mattermost_Community-cloud_channel-blue.svg)](https://community.mattermost.com/core/channels/cloud)
-Mattermost Operator for Kubernetes is under construction.
 
 ## Status
-
-The Mattermost operator is in alpha, data loss might occur.
+The Mattermost operator is in alpha but being actively developed.
 
 ## Summary
-
 Mattermost is a scalable, open source collaboration tool. It's written in Golang and React.
 
 This project offers a Kubernetes Operator for Mattermost to simplify deploying and managing your Mattermost instance.
 
+Learn more about Mattermost at https://mattermost.com.
 
-## Getting Started
-
-### Prerequisites
+The Mattermost server source code is available at https://github.com/mattermost/mattermost-server.
 
-- Kubernetes cluster 1.8.0+.
-- `kubectl` configured for the relevant Kubernetes cluster. (https://kubernetes.io/docs/reference/kubectl/overview/)
+## 1. Prerequisites
 
-### Install Operators
+### 1.1 Kubernetes Cluster with `kubectl`
+You must have a running Kubernetes 1.8.0+ cluster. If you do not, then see the [official Kubernetes documentation](https://kubernetes.io/docs/setup/) to get one set up.
```

Comment:
nit: I don't think we have actively tested anything below 1.11.X, so we may want to bump this number up unless you have tried something older.